--- /usr/share/perl5/Firefox/Marionette.pm	2025-03-09 17:00:42.000000000 -0400
+++ /home/shalaev/tmp/m.pm	2025-12-25 21:33:52.286609023 -0500
@@ -4701,6 +4701,10 @@
     my ( $self, %parameters ) = @_;
     my @arguments = qw(-marionette);
 
+    if ($self->_is_firefox_major_version_at_least(138)) {
+        push @arguments, '-remote-allow-system-access';
+    }
+
     if ( defined $self->{window_width} ) {
         push @arguments, '-width', $self->{window_width};
     }
